<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AstraAppClass</class>
 <widget class="QMainWindow" name="AstraAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstraApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Patient Number:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>1141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>Please select mode:</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Back_120_ArmAside</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Back_120_ArmAside_6points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Sagittal_L_120cm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Sagittal_R_120cm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Back_120_ArmsUp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Right_120_3side_points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Back_120_6side_points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>90</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Left_120_3side_points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>a_feet1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_10">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>a_feet2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_11">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>100</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>a_feet3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_12">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>60</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>b_feet2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_13">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>100</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>b_feet3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_14">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>20</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>b_feet1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_15">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>60</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>walk_2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_16">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>100</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>walk_3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_17">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>20</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15px &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>walk_1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>691</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Save Location: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>570</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Capture &amp;&amp; Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Show in Explorer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AstraApp.qrc"/>
 </resources>
 <connections/>
</ui>
